### PR TITLE
Allows wearing belts without requiring a uniform

### PR DESCRIPTION
**Pull request by ghost**
[tweak]
This enables the wearing of belts without requiring a uniform. Non-belt belt slot items still require a jumpsuit. 

:cl:
 * tweak: You can now wear belts without a uniform.

### CODE REVIEW

**Comment by Intigracy** [comment]
What happens if you pull your jumpsuit off while wearing one?

**Comment by ghost** [comment]
I was hoping nobody would ask that but I fixed it so it won't unequip your belt.

**Comment by kilozombie** [comment]
Taking someone's full-body jumpsuit off without taking off their belt seems like a very difficult task, and adds another thing you have to click on when stripping someone, for no apparent reason.

**Comment by nuklearcellphoneg** [comment]
You are strangling my immersions how could you betray me as such

**Comment by ghost** [comment]
I was going to use immulsions as an excuse for being lazy, but you can switch equipment items without removing them first so that's not really valid tbh

**Comment by RealestEstate** [comment]
hardcore security and SPS buff right here

**Comment by ghost** [comment]
> 
> 
> hardcore security and SPS buff right here

How so?

**Comment by RealestEstate** [comment]
iirc stripping jumpsuits removed the belt which didn't always trigger SPS on the belt.

**Comment by PJB3005** [comment]
Thats sounds like a bug that should be fixed anyways.

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 1]
```diff
@@ -255,7 +255,7 @@
 			u_equip(l_store, 1)
 		if (wear_id)
 			u_equip(wear_id, 1)
-		if (belt)
+		if (belt && !istype(belt, /obj/item/weapon/storage/belt))
```
Comment (PJB3005) [inline code/modules/mob/living/carbon/human/inventory.dm]: This check won't work correctly with the red ribbon arm, which sure looks like a belt to me.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/inventory.dm (hunk 1 above)
Comment (ghost) [reply code/modules/mob/living/carbon/human/inventory.dm]: OK, I had to add some other stuff but red ribbon arm behaves properly now.

**Comment by ShiftyRail** [comment]
This bug was fixed a few months back iirc

#19554
#19596

(actually, it was more than a year ago)

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**